### ISSUE TITLE
Value truncated error

### ISSUE THREAD

**Issue by AaronSadlerUK** [UmbHost/UmbCheckout#37]
When adding a license key it can throw the following exception:

```
Microsoft.Data.SqlClient.SqlException (0x80131904): String or binary data would be truncated in table '.dbo.umbracoKeyValue', column 'value'. Truncated value: ''.
The statement has been terminated.
```

**Comment by AaronSadlerUK** [comment]
Fixed in 1.0.1

**Comment by AaronSadlerUK** [comment]
This can use the original KeyVault table in the next LTS

https://github.com/umbraco/Umbraco-CMS/pull/14085
